Formula/tarantool.rb

```diff
@@ -23,7 +23,10 @@ class Tarantool < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "autoconf"
+  depends_on "automake"
```

Comment:
Can we use the `libunwind` formula instead of building another one?

Reply:
Unfortunately, it doesn't work. If you change this to `depends_on "libunwind"`, you will get this:
```console
cd /tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/third_party/libunwind && autoreconf -i
cd /tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/src && /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/gcc-13 -DCORO_ASM -DDT_NAMESPACE=tnt_ -DLUAJIT_SMART_STRINGS=1 -DNDEBUG -DNVALGRIND -DNVALGRIND=1 -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS=1 -D__STDC_FORMAT_MACROS=1 -D__STDC_LIMIT_MACROS=1 -I/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/src -I/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/src/lib -I/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/src/lib/small/include -I/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/src/lib/small/third_party -I/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/src/lib/core -I/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/third_party -I/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/third_party/xxHash -I/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/third_party/luajit/src -I/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/third_party/coro -I/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/src/lib/msgpuck -I/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/third_party/decNumber -I/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/third_party/libutil_freebsd -isystem /tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/third_party/libunwind/include -fexceptions -funwind-tables -fasynchronous-unwind-tables -fno-common -fopenmp -msse2 -Wformat -Wformat-security -Werror=format-security -fstack-protector-strong -fPIC -fmacro-prefix-map=/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0=. -std=c11 -Wall -Wextra -Wno-gnu-alignof-expression -fno-gnu89-inline -Wno-cast-function-type -O3 -DNDEBUG -MD -MT src/CMakeFiles/cpu_feature.dir/cpu_feature.c.o -MF CMakeFiles/cpu_feature.dir/cpu_feature.c.o.d -o CMakeFiles/cpu_feature.dir/cpu_feature.c.o -c /tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0/src/cpu_feature.c
/bin/sh: 1: autoreconf: not found
make[2]: *** [CMakeFiles/libunwind-build.dir/build.make:105: third_party/libunwind/configure] Error 127
make[2]: Leaving directory '/tmp/tarantool-20230525-3753943-x9mhuz/tarantool-2.11.0'
make[1]: *** [CMakeFiles/Makefile2:1778: CMakeFiles/libunwind-build.dir/all] Error 2
```

We need `autoconf`, `automake`, `libtool`.

Reply:
yeah, that is what we saw, we probably should file a ticket or patch to the upstream before merging your change

Reply:
Could you please explain what's wrong with adding `autoconf`, `automake`, `libtool` as dependencies?

Reply:
Because this should use Homebrew `linbunwind` instead of building its own. It already vendors its own LuaJIT, let's not add to that. See https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae.

Reply:
Also, `autoconf`, `automake`, and `libtool` aren't even needed on macOS, so these shouldn't be unconditional dependencies either.

Reply:
Got you, thanks! Fixed.